client/components/profilePage/myLibrary.jsx

```diff
@@ -1,75 +1,63 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import { Button, ListGroup, Container } from 'react-bootstrap';
 import BookDetail from '../global/BookDetail.jsx';
 import RecommendedBooks from './recommendedBooks.jsx';
 import 'bootstrap/dist/css/bootstrap.min.css';
+import inLibraryMark from './book.png';
 
-export default function myLibrary() {
-  const books = [
-    {
-      title: 'Green Eggs & Ham',
-      authors: ['Dr. Seuss'],
-      isbn: 7891036757892,
-      description: 'I do not like my green eggs & ham, Sam I am',
-      isbn: '9780375850967',
-      image:
-        'http://books.google.com/books/content?id=h7w4DwAAQBAJ&printsec=frontcover&img=1&zoom=5&edge=curl&source=gbs_api',
-      price: '$13.99',
-      category: 'Juvenile Fiction',
-      rating: 3,
-      ratingCount: 11,
-      datePurchased: '11/30/2021'
-    },
-    {
-      title: 'One Fish, Two Fish',
-      authors: ['Dr. Seuss'],
-      isbn: 7891036457892,
-      description: 'One Fish, Two Fish, Red Fish, Blue Fish',
-      isbn: '5760375843767',
-      image:
-        'http://books.google.com/books/content?id=067xAwAAQBAJ&printsec=frontcover&img=1&zoom=5&edge=curl&source=gbs_api',
-      price: '$19.99',
-      category: 'Juvenile Fiction',
-      rating: 5,
-      ratingCount: 7,
-      datePurchased: '02/15/2021'
-    }
-  ];
-
-  const [booksOwned, setBooksOwned] = useState(books);
+export default function myLibrary({ user }) {
+  const [booksOwned, setBooksOwned] = useState([]);
   const [searchInput, setSearchInput] = useState('');
   const [clickedBook, setClickedBook] = useState();
```

Comment:
Don't know if there is gonna be a bug is you don't set initial value of clickedBook, maybe more safe to do `userState({})`

Reply:
Debugging some interactions with the new book details and that is actually a change I made. Good catch.